pretyping/tacred.ml

```diff
@@ -656,11 +657,29 @@ let whd_nothing_for_iota env sigma s =
   in
   whrec s
 
+(* The reductions that should be performed as part of the simpl tactic,
+  excluding symbols that have the NeverUnfold flag. *)
+
+let make_simpl_reds () =
+  let open CClosure.RedFlags in
+  red_add
+    (red_add
+       (GlobRef.Set.fold
+          (fun x r ->
+            red_sub r
+              (match x with
+                 Names.GlobRef.ConstRef xx ->  fCONST xx
+               | _ -> assert false))
```

Comment:
```suggestion
              (match x with
                 Names.GlobRef.ConstRef xx ->  red_sub r (fCONST xx)
               | _ -> r))
```
This way there is no `assert false` (the data structures are about any globref, I don't even know if it is true that you cannot make an inductive simpl never even if it would have no effect).